new_webapp/src/index.css

```diff
@@ -1,17 +1,40 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
 
 .badge {
   float: left
+}
+
+.hplus_font {
+    text-align: left;
+    margin-top: 40px;
+}
+
+.title {
+    text-align: left;
+    margin-top: -10px;
+}
+
+.description {
+    text-align: left;
+    margin-top: 15px;
+}
+
+.col-10 {
```

Comment:
This seems kinda dangerous to me, `.col-10` is a grid modifier in bootstrap. Can you not accomplish this with a custom class or with `justify-content-center` on the div with the `row` class?

I'm afraid of this causing headaches later or on other parts of the UI that have col-10.

Reply:
Sorry, this was an old change. I meant to remove it, but had forgotten!